packages/manager/modules/netapp/src/onboarding/constants.js

```diff
@@ -1,3 +1,117 @@
-export const GUIDES = [];
+export const GUIDES = [
+  {
+    id: 'manager_start',
+    highlight: true,
+    link: {
+      ASIA:
+        'https://docs.ovh.com/asia/en/storage/netapp-managing-control-panel/',
+      AU: 'https://docs.ovh.com/au/en/storage/netapp-managing-control-panel/',
+      CA: 'https://docs.ovh.com/ca/en/storage/netapp-managing-control-panel/',
+      DE: 'https://docs.ovh.com/de/storage/netapp-managing-control-panel/',
+      ES: 'https://docs.ovh.com/es/storage/netapp-managing-control-panel/',
+      FR: 'https://docs.ovh.com/fr/storage/netapp-managing-control-panel/',
+      GB: 'https://docs.ovh.com/gb/en/storage/netapp-managing-control-panel/',
+      IE: 'https://docs.ovh.com/ie/en/storage/netapp-managing-control-panel/',
+      IT: 'https://docs.ovh.com/it/storage/netapp-managing-control-panel/',
+      PL: 'https://docs.ovh.com/pl/storage/netapp-managing-control-panel/',
+      PT: 'https://docs.ovh.com/pt/storage/netapp-managing-control-panel/',
+      QC: 'https://docs.ovh.com/ca/fr/storage/netapp-managing-control-panel/',
+      SG: 'https://docs.ovh.com/sg/en/storage/netapp-managing-control-panel/',
+      US: 'https://docs.ovh.com/us/en/storage/netapp-managing-control-panel/',
+      WS: 'https://docs.ovh.com/us/es/storage/netapp-managing-control-panel/',
```

Comment:
no default guide URL? its good to have a default guide URL. Same commend for all URL's